### ISSUE TITLE
Add `aria-label` requirement for `HiddenTextExpander`

### ISSUE THREAD

**Issue by khiga8** [primer/view_components#637]
Currently the button contains a hellip so it's not being flagged by our axe checks, but we should require an `aria-label` pertaining to what is collapsed/expanded.

Additional thoughts: should this be a behavioral component given it's associated with js?

**Comment by Artemisia-Absinthium** [comment]
Trying to understand it more, I looked at previous discussions and found this one: https://github.com/primer/view_components/pull/446/files#r612735161

Looking at it, my understanding is that `HiddenTextExpander` is not associated with any JS yet (Or I missed it) and that we might have another component, a behavioural one this time, handling the collapse/expend feature. 
Is it the case?

@manuelpuyol , was this discussed again?

Also, in my opinion, `HiddenTextExpander` is conveying that functionality in the name. Should we rename it to something like `HellipButton` if we have that bigger component with the JS behaviour?

I will add a required label to that button anyway since it will be needed. :)

**Comment by manuelpuyol** [comment]
I'm 👍  on renaming that button!

We haven't discussed the behavioral component yet, maybe we can investigate how dotcom applies behaviors and try to mimic it here

**Comment by Artemisia-Absinthium** [comment]
I would be up for it as I also want to learn how to set up behavioural component.

And I will rename it then. :)
